docs/spec/sections/authorization.html

```diff
@@ -29,18 +29,17 @@ <h2>Authentication</h2>
   <section id="presentation-authentication">
     <h3>Presentation Authentication</h3>
     <p>
-      The following sequence of events is necessary for a holder to present data to a verifier:
+      The following sequence of events is necessary for a Holder to present data to a Verifier:
     </p>
     <ul>
       <li>
-        The verifier MUST establish the <a href="https://www.rfc-editor.org/rfc/rfc6749.html">OAuth 2.0</a> binding by
-        providing client ID and secret to the holder.
+        The Holder MUST authenticate by obtaining an
```

Comment:
I think you have a mixup with the parties in this sentence

Reply:
Ah right! Good catch - thanks!